src/audio/voice/capture/voice_capture.cpp

```diff
@@ -0,0 +1,165 @@
+#include "voice_capture.hpp"
+#include "constants.hpp"
+
+namespace AbaddonClient::Audio::Voice {
+
+using OpusEncoder = Opus::OpusEncoder;
+using EncoderSettings = OpusEncoder::EncoderSettings;
+using SignalHint = OpusEncoder::SignalHint;
+using EncodingApplication = OpusEncoder::EncodingApplication;
+
+void capture_callback(ma_device* pDevice, void* pOutput, const void* pInput, ma_uint32 frameCount) {
+    auto capture = reinterpret_cast<VoiceCapture*>(pDevice->pUserData);
```

Comment:
you probably got this from me cuz i didnt know but this can be a static_cast

Reply:
> you probably got this from me

Yes I did